tenant_schemas/apps.py

```diff
@@ -42,7 +42,8 @@ def best_practice(app_configs, **kwargs):
 
     errors = []
 
-    if INSTALLED_APPS[0] != 'tenant_schemas':
+    django_index = next(i for i, s in enumerate(INSTALLED_APPS) if 'django.' in s)
```

Comment:
Using `in` will result in a match of `thirdparty.django.blah` - might be better to use `s.startswith('django.')` instead.